OKR.md

```diff
@@ -0,0 +1,31 @@
+# Quarterly Objectives and Key Results
+
+We try to frame our ongoing work using a process based on quarterly Objectives and Key Results (OKRs). Objectives reflect outcomes that are challenging, but realistic. Results are tangible and measurable.
+
+## 2018 Q3
+
+**All pieces are there. Ship Peer & Content Routing + IPNS**
+
+- Delegated Peer Routing is completed 
+- Delegated Content Routing is completed  
+- DHT interops with go-libp2p
+- IPNS works over PubSub (+ go-libp2p interop) - @vascosantos
+- IPNS works over DHT (+ go-libp2p interop) - @vascosantos
```

Comment:
IPNS is currently implemented in `go-ipfs`, so I think we should have `go-ipfs interop`

Reply:
<img width="815" alt="image" src="https://user-images.githubusercontent.com/1211152/42269438-6e0128da-7f7e-11e8-8670-c2bb4a3a069c.png">

You mean anything different from what is already there?

Reply:
I mean:

```
IPNS works over PubSub (+ go-ipfs interop) - @vascosantos
IPNS works over DHT (+ go-ipfs interop) - @vascosantos
```

Since IPNS is implemented inside the scope of `js-ipfs` / `go-ipfs`